internal/monitors/expvar/expvar.go

```diff
@@ -140,7 +141,9 @@ func (m *Monitor) fetchMetrics() (map[string][]*datapoint.Datapoint, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer resp.Body.Close()
+	defer func() {
```

Comment:
Why not `if res != nil { defer res.Body.Close() }` instead, unless you want to log the error returned by `res.Body.Close()`

Reply:
I just changed it to ignore the return value of `Close()` to get rid of a warning about unchecked err. It _should_ be the same behavior as before but let me know if I'm missing something.

Reply:
The only potential difference between the old and new is that I think it evaluates the value of `resp.Body` when registering the defer in the old way and evaluates it after function return in the anonymous func way.  Since `resp` and `resp.Body` aren't getting reassigned later in the function it doesn't make any difference here.